artifactory_cleanup/artifactorycleanup.py

```diff
@@ -1,19 +1,20 @@
 from concurrent.futures import ThreadPoolExecutor
 from datetime import date
-from typing import List, Iterator
+from typing import List, Iterator, Optional
 
 from attr import dataclass
 from requests import Session
 
 from artifactory_cleanup.errors import ArtifactoryCleanupException
-from artifactory_cleanup.rules.base import CleanupPolicy, ArtifactDict
+from artifactory_cleanup.rules.base import ArtifactsList, CleanupPolicy, ArtifactDict
 
 
 @dataclass
 class CleanupSummary:
     policy_name: str
     artifacts_removed: int
     artifacts_size: int
+    removed_artifacts_list: Optional[ArtifactsList] = None
```

Comment:
```suggestion
    removed_artifacts: ArtifactsList = None
```

Reply:
Good suggestions, latest fixup contains:

- I renamed `removed_artifacts` to `artifacts` because we are already in the context of CleanupSummary
- the json output to a file is now indented 
- clean up of `artifactorycleanup.py` 
- description of `--output-artifacts` updated